SYCL/Basic/fpga_tests/fpga_dsp_control.cpp

```diff
@@ -0,0 +1,65 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUNx: %ACC_RUN_PLACEHOLDER %t.out
```

Comment:
Why run is disabled?

Reply:
I copied the run strings directly from the existing test fpga_lsu.cpp. Didn't realize "RUNx" means the run is disabled.
I can remove the "x" and test again.

Reply:
Hw, @mlychkov do you know, why fpga_lsu is disabled?

Reply:
I think one comment in fpga_lsu.cpp explained:
```
// TODO: run is disabled, since no support added in FPGA backend yet. Check
// implementation correctness from CXX and SYCL languages perspective.
```

Reply:
@shuoniu-intel Yes, it would be nice if you enabled fpga_lsu.cpp test as well.